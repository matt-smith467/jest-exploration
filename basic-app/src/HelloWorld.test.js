import React from "react";
import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import HelloWorld from "./components/HelloWorld.js";

test('renders the "Hello World" message', () => {
  render(<HelloWorld />);
  const helloWorldText = screen.getByText("Hello World");
  expect(helloWorldText).toBeInTheDocument();
});

it("performs snapshot testing", () => {
  const tree = renderer.create(<HelloWorld />).toJSON();
  expect(tree).toMatchSnapshot();
});
