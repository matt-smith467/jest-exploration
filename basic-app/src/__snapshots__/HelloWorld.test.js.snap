// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`performs snapshot testing 1`] = `
<div>
  <h1>
    Hello World
  </h1>
</div>
`;
